kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -709,31 +733,129 @@ public Properties getOriginalProperties() {
     return properties;
   }
 
+  public final Map<String, Object> getSchemaRegistryConfigs() {
+    Set<String> mask = AbstractKafkaSchemaSerDeConfig.baseConfigDef().names();
+    Map<String, Object> configs =
+        new HashMap<>(
+            new ConfigsBuilder(mask)
+                .addConfigs("schema.registry.", /* strip= */ false)
```

Comment:
is this because of `schema.registry.url`? kinda sucks, i guess this is fine.

Reply:
it might be worth commenting on the reason.

Reply:
Done. Also restricted this to only `schema.registry.url` to avoid catching things we don't want to.